src/main/java/com/google/sps/classes/UserAuthenticationData.java

```diff
@@ -20,12 +20,14 @@
 public final class UserAuthenticationData {
 
   private final boolean loggedIn;
-  private final String authenticationUrl; // URL to redirect to to login or logout page.
   private final String email;
+  private final boolean isUserRegistered;
+  private final String authenticationUrl;
```

Comment:
Can you keep the comment here for clarity?

Reply:
Done!